util/chunk/chunk.go

```diff
@@ -30,24 +32,69 @@ type Chunk struct {
 	// numVirtualRows indicates the number of virtual rows, which have zero column.
 	// It is used only when this Chunk doesn't hold any data, i.e. "len(columns)==0".
 	numVirtualRows int
+	// capacity indicates the max number of rows that the chunk wants to hold.
+	// The accurate row count after executor.Next is guaranteed by executor implement.
+	// It is used only during executor's execution, and grow by `Renew` or `Reset`.
```

Comment:
1. `grow` or `change`?
2. only `Renew` now?

Reply:
1. `grow`
2. oh now is `GrowThenReset` but after continue pr will be `Reset`